svo (install me in module manager).xml

```diff
@@ -7272,13 +7271,13 @@ end</script>
 	-- sync this module too
 	if syncModule then syncModule('svo (install me in module manager)') end
 end
-svo.install_other_modules()
```

Comment:
This line would never do anything, as the function will only complete when passed a modulename as the second argument.